### DISCUSSION TITLE
mitmdump 6.0.2 with pyOpenSSL latest

### DISCUSSION THREAD

**Discussion by mrsiano** [mitmproxy/mitmproxy#5239] [General]
Hi Guys,
Is that possible to ask please to build mitmproxy 6.0.2 with latest version of pyOpenSSL.

also, you have some docs of how to build the project from source ?
@mhils

**Comment by mhils** [comment]
See our supported version policy at https://github.com/mitmproxy/mitmproxy/blob/main/SECURITY.md#supported-versions. 
Only the most recent release is supported.

If you want to build custom binaries, take a look at https://github.com/mitmproxy/mitmproxy/blob/85e57a91e0cc96affc282d511ce606782ea2047b/.github/workflows/main.yml#L97-L133. Most of the stuff is in https://github.com/mitmproxy/mitmproxy/blob/main/release/cibuild.py. :)
